cdap-explore/src/main/java/co/cask/cdap/explore/executor/ExploreExecutorHttpHandler.java

```diff
@@ -511,6 +518,19 @@ private static String isClassNotFoundException(Throwable e) {
     return null;
   }
 
+  @Nullable
```

Comment:
Should these be done as @HeaderParam? What is the advantage of extracting them manually?